template.yml

```diff
@@ -33,3 +37,11 @@ Resources:
           Type: Schedule
           Properties:
             Schedule: rate(1 hour)
+
+Parameters:
+  VersionsToKeep:
+    Type: Number    
+    Description: >
```

Comment:
nit: Use `>-` instead of `>` or else you end up with a trailing newline on your multiline comment. This trailing newline can be annoying when someone uses the "Copy as SAM resource" button in the Lambda Console to copy the code needed to nest the app in their SAM template.

Reply:
thanks! that's a good tip